build/js/verb.js

```diff
@@ -3846,7 +3846,7 @@ verb_eval_Analyze.rationalCurveClosestParam = function(curve,p) {
 		cu = ct;
 		i++;
 	}
-	return cu;
+	return u;
```

Comment:
Not sure if I understood this change. `cu` is updated inside the while loop but will be ignored by returning `u`?

Reply:
Yes, it gets there only if it does not converge, returning u computed as a closest tessellated point

Reply:
oh I see, `cu` is returned inside the while loop if certain condition is met, but if not, it will return the initial value...